src/Components/Components/src/Microsoft.AspNetCore.Components.csproj

```diff
@@ -62,4 +57,8 @@
     <NuspecProperty Include="PackageThirdPartyNoticesFile=$(PackageThirdPartyNoticesFile)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\JSInterop\Microsoft.JSInterop\src\Microsoft.JSInterop.csproj" />
```

Comment:
Doesn't this project already reference JSInterop? You shouldn't need to do add this reference

Reply:
Nope, see comment above.

Reply:
It appears not. @SteveSandersonMS mentioned above that it would be best not to reference JS interop from the `Components` package, so I'll have to look into another solution. I can try changing `ElementReference` from having an `IJSInterop` property to having an `IServiceProvider` property instead, although that might get tricky with how `ElementReference`s are currently serialized/deserialized using `ElementReferenceJsonConverter`.